### DISCUSSION TITLE
docs | What technology stack was used to develop the document?

### DISCUSSION THREAD

**Discussion by yayxs** [pmndrs/zustand#2224] [Q&A]
When I first saw the document, I thought it was very simple

this document https://docs.pmnd.rs/zustand/getting-started/introduction

I looked through it and found this git repository https://github.com/pmndrs/website , but it doesn't seem to be

Where is the source code of the official document, which can be used for learning?

Thanks

**Comment by dai-shi** [comment]
See https://github.com/pmndrs/website/tree/docs
